crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -3073,4 +3078,50 @@ mod tests {
 
         Ok(())
     }
+
+    #[async_test]
+    async fn test_sync_beat_is_notified_on_sync_response() -> Result<()> {
+        let server = MockServer::start().await;
+        let client = logged_in_client(Some(server.uri())).await;
+
+        let pos = Arc::new(Mutex::new(0));
+        let _mock_guard = Mock::given(SlidingSyncMatcher)
+            .respond_with(move |_: &Request| {
+                let mut pos = pos.lock().unwrap();
+                *pos += 1;
```

Comment:
You don't need the whole `pos` stuff, since it's only returning once, you can return an hardcoded pos of `1`?